fi.json

```diff
@@ -1009,17 +1009,17 @@
             "option-backlink-in-document-desc": "Show backlinks in document by default when opening new panes."
         },
         "outgoing-links": {
-            "name": "Outgoing Links",
-            "desc": "Show outgoing links and detect unlinked mentions of other notes in the current note.",
-            "action-open": "Open outgoing links",
-            "action-show": "Show outgoing links pane",
-            "action-open-for-current": "Open outgoing links for the current file",
-            "tab-title": "Outgoing links from {{displayText}}",
-            "label-links": "Links",
-            "label-no-links": "No links found.",
-            "label-unlinked-mentions": "Unlinked mentions",
-            "tooltip-link-file": "Link this file",
-            "tooltip-not-created": "Not created yet"
+            "name": "Ulosmenevät Linkit",
+            "desc": "Näytä ulosmenevät linkit ja tunnista muiden muistiinpanojen linkittämättömät maininnat nykyisessä muistiinpanossa.",
+            "action-open": "Avaa ulosmenevät linkit",
+            "action-show": "Näytä ulosmenevien linkkien paneeli",
+            "action-open-for-current": "Avaa nykyisen tiedoston ulosmenevät linkit",
+            "tab-title": "Ulosmenevät linkit {{displayText}}:sta",
```

Comment:
This is a bit janky, but I couldn't think of a better way. I couldn't find any context for that, but maybe `"Ulosmenevät linkit kohteesta {{displayText}}"` would be better?